axum/src/extract/matched_path.rs

```diff
@@ -143,7 +143,6 @@ pub(crate) fn set_matched_path_for_request(
 
     if matched_path.ends_with(NEST_TAIL_PARAM_CAPTURE) {
         extensions.insert(MatchedNestedPath(matched_path));
-        debug_assert!(matches!(dbg!(extensions.remove::<MatchedPath>()), None));
```

Comment:
We still want the assert, just not the `dbg!`

Reply:
Oh, fixed